Assignment2_q1.py

```diff
@@ -0,0 +1,80 @@
+# coding=utf-8
+from BinaryTree import TreeNode, BinaryTree
+
+"""Prints all the ancestors of the key in the given binary tree. Uses two methods:
```

Comment:
Changed now! Thanks for your advice, I didn't know how to change a message in the description field in interactive shell.